test/FunctionsV2/CorrelationEndToEndTests.cs

```diff
@@ -219,109 +219,6 @@ internal async Task<Tuple<List<OperationTelemetry>, List<ExceptionTelemetry>>>
             return new Tuple<List<OperationTelemetry>, List<ExceptionTelemetry>>(result.CorrelationSort(), exceptionTelemetryList);
         }
 
-         /*
-         * End to end test that checks if a warning is logged when distributed tracing is
-         * enabled, but APPINSIGHTS_INSTRUMENTATIONKEY isn't set. The test also checks
-         * that the warning isn't logged when the environment variable is set.
-         */
-        [Theory]
-        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
-        [InlineData(false, false, false)]
-        [InlineData(false, false, true)]
-        [InlineData(true, false, false)]
-        [InlineData(true, false, true)]
-        [InlineData(false, true, false)]
-        [InlineData(false, true, true)]
-        [InlineData(true, true, false)]
-        [InlineData(true, true, true)]
-        public void TelemetryClientSetup_AppInsights_Warnings(bool instrumentationKeyIsSet, bool connStringIsSet, bool extendedSessions)
```

Comment:
Good call deleting this. Not something durable should be enforcing - particularly with OTel coming along which will not always be app-insights backed.